credentials/build.gradle.kts

```diff
@@ -11,31 +11,43 @@ repositories {
   maven("https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/")
 }
 
-val ktor_version = "2.3.4"
-val jackson_version = "2.14.2"
-
 dependencies {
-  api("com.danubetech:verifiable-credentials-java:1.6.0")
-
+  /**
+   * Maintainers - please do not declare versioning here at the module level;
+   * versioning is centralized for the platform in $projectRoot/gradle/libs.versions.toml
+   *
+   * Deps are declared in alphabetical order.
+   */
+
+  // API
+  /*
+   * API Leak: https://github.com/TBD54566975/web5-kt/issues/228
+   *
+   * Change and move to "implementation" when completed
+   */
+  api(libs.comDanubetechVerifiableCredentials)
+
+  // Project
   implementation(project(":dids"))
   implementation(project(":common"))
   implementation(project(":crypto"))
-  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jackson_version")
-  implementation("com.nfeld.jsonpathkt:jsonpathkt:2.0.1")
-  implementation("com.nimbusds:nimbus-jose-jwt:9.34")
-  implementation("decentralized-identity:did-common-java:1.9.0")
-  implementation("com.networknt:json-schema-validator:1.0.87")
-
-  implementation("io.ktor:ktor-client-core:$ktor_version")
-  implementation("io.ktor:ktor-client-okhttp:$ktor_version")
-  implementation("io.ktor:ktor-client-content-negotiation:$ktor_version")
-  implementation("io.ktor:ktor-serialization-jackson:$ktor_version")
-  implementation("io.ktor:ktor-serialization-kotlinx-json:$ktor_version")
-  implementation("io.ktor:ktor-client-logging:$ktor_version")
-
-  testImplementation("io.ktor:ktor-client-mock:$ktor_version")
 
+  // Implementation
+  implementation(libs.comFasterXmlJacksonModuleKotlin)
+  implementation(libs.comNetworkntJsonSchemaValidator)
+  implementation(libs.comNfeldJsonpathkt)
+  implementation(libs.comNimbusdsJoseJwt)
+  implementation(libs.decentralizedIdentityDidCommonJava)
+  implementation(libs.bundles.ioKtorForCredentials)
+
+  // Test
+  /**
+   * Test dependencies may declare direct versions; they are not exported
+   * and therefore are within the remit of this module to self-define
+   * if desired.
+   */
+  testImplementation(libs.ioKtorClientMock)
   testImplementation(kotlin("test"))
-  testImplementation("com.willowtreeapps.assertk:assertk:0.27.0")
+  testImplementation(libs.comWillowtreeappsAssertk)
```

Comment:
Is the more object-y import better than the string-y import for a specific reason or is it a style thing?

Reply:
i _think_ this is referenced via `gradle/libs.versions.toml` ? then we can use the same object-y import (with same version) for each subproject (if we use it in more than one subproject)

Reply:
Exactly, @jiyoontbd! Only reason I didn't enforce the same objectey format in test deps throughout the subprojects is because test deps don't *need* to be aligned among modules; they're not exported to the compile or runtime classpaths of consuming projects.